src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4695,12 +4695,7 @@ namespace VariableSpeedCoils {
                         SupTemp -= FanCoolLoad / (CpAir * rhoair * VolFlowRate);
                     }
                     MixWetBulb = Psychrometrics::PsyTwbFnTdbWPb(state, MixTemp, MixHumRat, state.dataEnvrn->StdBaroPress, RoutineName);
-                    // need to use OutTemp for air-cooled and RatedInletWaterTemp for water-cooled
-                    if (varSpeedCoil.CondenserInletNodeNum != 0) {
-                        RatedSourceTempCool = RatedInletWaterTemp;
-                    } else {
-                        RatedSourceTempCool = OutTemp;
-                    }
+                    RatedSourceTempCool = GetVSCoilRatedSourceTemp(state, DXCoilNum, ErrorsFound);
```

Comment:
This block of code and the next one will create diffs. Previously, only coils with a condenser inlet node input were assigned `RatedInletWaterTemp` (29.4444C) as `RatedSourceTempCool`. `Coil:Cooling:WaterToAirEquationFitVariableSpeed` objects don't have such inputs so their `RatedInletWaterTemp` was set to `OutTemp`.